doc/user/release-notes.html

```diff
@@ -56,6 +56,8 @@ <h2>Release notes for Crux</h2>
 
 <h4>Minor changes</h4>
 <ul>
+  <li>13 Jan 2023: tide-search halts when tailor-calibration is intented to be 
```

Comment:
"halts with an error message when the user tries to use tailor-calibration in conjunction with peptide-centric search."